### ISSUE TITLE
Feature Request: support `xccov` JSON format

### ISSUE THREAD

**Issue by eelco** [codeclimate/test-reporter#377]
For a while now (since Xcode 9.3) Apple ships a tool for code coverage. (Run it with `xcrun xccov`).

It can read the coverage files generated by Xcode and translate them into JSON.

It would be great if the test reporter could support those JSON files, as it would save a bunch of dependencies for getting Objective-C / Swift coverage into Code Climate.

**Comment by wfleming** [comment]
Good idea! Our team probably won't be able to incorporate this into our own planning in the near-term, so I've added the "help wanted" label here. This would be great to see as a PR if anyone has time.

**Comment by andresilveirah** [comment]
Sorry for the digging this issue out of the grave but has this perhaps been addressed on #399 ? And could anyone point me to the doc on how to use it?

**Comment by manicmaniac** [comment]
I also cannot find the document but it worked as I expected.

```sh
$ xcodebuild test -resultBundlePath result_bundle.xcresult
$ xcrun xccov view --report --json result_bundle.xcresult > coverage.json
$ cc-test-reporter format-coverage -t xccov coverage.json
```
